apps/ui/src/networks/evm/index.ts

```diff
@@ -72,6 +72,24 @@ export const METADATA: Record<string, Metadata> = {
       'ipfs://bafkreid7ndxh6y2ljw2jhbisodiyrhcy2udvnwqgon5wgells3kh4si5z4',
     blockTime: ETH_MAINNET_BLOCK_TIME
   },
+  ape: {
+    name: 'ApeChain',
+    chainId: 33139,
+    currentChainId: 1,
+    apiUrl: UNIFIED_API_URL,
+    avatar:
+      'ipfs://bafkreielbgcox2jsw3g6pqulqb7pyjgx7czjt6ahnibihaij6lozoy53w4',
+    blockTime: ETH_MAINNET_BLOCK_TIME
+  },
+  curtis: {
+    name: 'Curtis',
+    chainId: 33111,
+    currentChainId: 11155111,
+    apiUrl: UNIFIED_API_TESTNET_URL,
+    avatar:
+      'ipfs://bafkreicljxttjq2xkgfwwpii5xegirgq2ctrnsjnzelxudjj33qzq65apu',
```

Comment:
We should use same logo than the mainnet
```suggestion
      'ipfs://bafkreielbgcox2jsw3g6pqulqb7pyjgx7czjt6ahnibihaij6lozoy53w4',
```